Index: Task_5.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Task_5.py	(revision 74939513237293105f179f022281d0ffa486e242)
+++ Task_5.py	(date 1586548790811)
@@ -15,12 +15,16 @@
         break
     else:
         print('Введите целое число')
-
-
 profit = revenue - costs
-
-employees = input("количество сотрудников")
-
+if profit < 0:
+    print("Фирма убыточна, нужно что-то менять")
+while True:
+    employees = input("количество сотрудников: ")
+    if employees.isdigit():
+        employees = int(employees)
+        break
+    else:
+         print('Введите целое число')
 profit_per_epml = profit / employees
 
-print("выручка на одного сотрудника составит {profit_per_epml} рублей")
\ No newline at end of file
+print(f"Выручка на одного сотрудника составит {profit_per_epml} рублей")
\ No newline at end of file
